src/node.cc

```diff
@@ -4092,7 +4087,13 @@ inline void PlatformInit() {
       ABORT();
   }
 
+#if HAVE_INSPECTOR
+  sigset_t sigmask;
+  sigemptyset(&sigmask);
+  sigaddset(&sigmask, SIGUSR1);
+  const int err = pthread_sigmask(SIG_SETMASK, &sigmask, nullptr);
   CHECK_EQ(err, 0);
+#endif  // HAVE_INSPECTOR
```

Comment:
Ah I see, that makes sense. I was trying to avoid the additional HAVE_INSTRUCTOR guard but that is no problem.

> I don't want to make this a torment

No torment at all, I appreciate all the reviews and feedback!